### ISSUE TITLE
Markdown formatting issues

### ISSUE THREAD

**Issue by davenicoll** [djsudduth/keep-it-markdown#23]
In a previous commit, line 254 was commented out -
```
#note_text = gnote.text #gnote.text.replace('”','"').replace('“','"').replace("‘","'").replace("’","'").replace('•', "-").replace(u"\u2610", '[ ]').replace(u"\u2611", '[x]').replace(u'\xa0', u' ').replace(u'\u2013', '--').replace(u'\u2014', '--').replace(u'\u2026', '...').replace(u'\u00b1', '+/-')
```
This, among other things, means that lists/checkboxes aren't properly converted to markdown. I had to uncomment this line to  format my notes correctly.

Was this commented in error?

**Comment by djsudduth** [comment]
There was a more generalized and cleaner formatting of note text added in 0.4.0 - checkboxes must have been missed. I will check into it.

**Comment by davenicoll** [comment]
Thanks @djsudduth ... FYI I only discovered after I posted this, that the markdown for a checkbox should be `- [ ]` or `- [x]` rather than just `[ ]`.

**Comment by djsudduth** [comment]
Glad you like it! I could not reproduce the problem. My keep check boxes and lists convert successfully and render markdown in Obsidian and Typora.  Do you have an example of how your note was formatted that you saw this issue?

**Comment by davenicoll** [comment]
Sure, here's a keep that tripped up.

It has an image.
```
☑ Complete Europe's longest zip line
☑ Visit 10 new national trusts
```
I'll do some more testing later today.

**Comment by davenicoll** [comment]
I don't see anywhere else in the code where text boxes are converted to their markdown equivalent, so I wondered if you had a version locally that you've not pushed? I've submitted a update to my PR which includes a fix for this too.

**Comment by djsudduth** [comment]
Thanks - that recreated the error - the LIST type is immutable for the gnote.text field thus throwing the error when attempting to prepend the image markdown (I haven't looked at your fix yet)

**Comment by djsudduth** [comment]
@davenicoll - could you take a look at this?  Sorry that I missed your pull request yesterday was the other bug and not this one ( I was in a hurry :) ).  I don't use lists much so this is probably why the output is incorrect.  Thanks for your help!

**Comment by djsudduth** [comment]
PS @davenicoll - the line that was commented out should be reduced to just checkboxes/list markdown. The original reason it was commented out was due to allowing full Unicode/foreign languages.  There may be some other markdown conversions necessary besides checkboxes.

**Comment by djsudduth** [comment]
@davenicoll - will you be taking a look at this? Or, do you want me to incorporate you other pull requests and manually merge it in?

**Comment by davenicoll** [comment]
Hey @djsudduth, sure, I'll take a look. I'll check the markdown formatting rules and see if anything should be in there, before submitting a PR.

**Comment by djsudduth** [comment]
I did the fix - in testing now

**Comment by djsudduth** [comment]
released in 0.4.2
